src/schemaText/bridge-text-en.json

```diff
@@ -1052,5 +1052,123 @@
+      "label": "Segment File Bytes"
+    },
+    "config_enable": {
+      "desc": "Enable (true) or disable (false) this Pulsar bridge.",
+      "label": "Enable or Disable"
+    },
+    "desc_name": {
+      "desc": "Bridge name, used as a human-readable description of the bridge.",
+      "label": "Bridge Name"
+    },
+    "desc_type": {
+      "desc": "The Bridge Type",
+      "label": "Bridge Type"
+    },
+    "batch_size": {
+      "desc": "Maximum number of individual requests to batch in a Pulsar message.",
+      "label": "Batch size"
+    },
+    "buffer": {
+      "desc": "Configure producer message buffer.\n\nTell Pulsar producer how to buffer messages when EMQX has more messages to send than Pulsar can keep up, or when Pulsar is down.",
+      "label": "Message Buffer"
+    },
+    "compression": {
+      "desc": "Compression method.",
+      "label": "Compression"
+    },
+    "message_key": {
+      "desc": "Template to render Pulsar message key.",
+      "label": "Message Key"
+    },
+    "local_topic": {
+      "desc": "MQTT topic or topic filter as data source (bridge input). If rule action is used as data source, this config should be left empty, otherwise messages will be duplicated in Pulsar.",
+      "label": "Source MQTT Topic"
+    },
+    "max_batch_bytes": {
+      "desc": "Maximum bytes to collect in a Pulsar message batch. Most of the Pulsar brokers default to a limit of 5 MB batch size. EMQX's default value is less than 5 MB in order to compensate Pulsar message encoding overheads (especially when each individual message is very small). When a single message is over the limit, it is still sent (as a single element batch).",
+      "label": "Max Batch Bytes"
+    },
+    "message_opts": {
+      "desc": "Template to render a Pulsar message.",
+      "label": "Pulsar Message Template"
+    },
+    "pulsar_message": {
+      "desc": "Template to render a Pulsar message.",
+      "label": "Pulsar Message Template"
+    },
+    "pulsar_topic": {
+      "desc": "Pulsar topic name",
+      "label": "Pulsar topic name"
+    },
+    "retention_period": {
+      "desc": "The amount of time messages will be buffered while there is no connection to the Pulsar broker.  Longer times mean that more memory/disk will be used",
+      "label": "Retention Period"
+    },
+    "send_buffer": {
+      "desc": "Fine tune the socket send buffer. The default value is tuned for high throughput.",
+      "label": "Socket Send Buffer Size"
+    },
+    "strategy": {
+      "desc": "Partition strategy is to tell the producer how to dispatch messages to Pulsar partitions.\n\n<code>random</code>: Randomly pick a partition for each message.\n<code>roundrobin</code>: Pick each available producer in turn for each message.\n<code>key_dispatch</code>: Hash Pulsar message key of the first message in a batch to a partition number.",
```

Comment:
```suggestion
      "desc": "Choose how messages are dispatched to Pulsar partitions:\n\n<code>random</code>: Messages are randomly assigned to partitions.\n<code>roundrobin</code>: Messages are evenly distributed across available producers.\n<code>key_dispatch</code>: Partitions to be selected are hashed as the Pulsar message key of the first message in a batch.",
```

Reply:
Partitions to be selected are hashed as the Pulsar message key of the first message in a batch.", this should be confirmed @wivwiv